cmd/erasure-server-pool.go

```diff
@@ -141,6 +141,10 @@ func newErasureServerPools(ctx context.Context, endpointServerPools EndpointServ
 	}
 
 	z.decommissionCancelers = make([]context.CancelFunc, len(z.serverPools))
+
+	// initialize the object layer.
+	setObjectLayer(z)
```

Comment:
Won't this change make the cluster crashes immediately when the server receives a request during startup ? no bucket metadata initializaed => crash ?

(do you want me to try that locallly ?)

Reply:
It will not crash @vadmeste 

- we freeze all S3 calls regardless of object layer init
- rest of the code always has fallback to read from disk